Cargo.toml

```diff
@@ -15,8 +15,17 @@ exclude = [
 ]
 
 [patch.crates-io]
-# tendermint              = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-rpc          = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-proto        = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-light-client = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
-# tendermint-testgen      = { git = "https://github.com/informalsystems/tendermint-rs", branch = "master" }
+# Uncomment the following lines to use dependencies from tendermint-rs on GitHub
+tendermint              = { git = "https://github.com/informalsystems/tendermint-rs", branch = "thane/ibc-1252" }
+tendermint-rpc          = { git = "https://github.com/informalsystems/tendermint-rs", branch = "thane/ibc-1252" }
+tendermint-proto        = { git = "https://github.com/informalsystems/tendermint-rs", branch = "thane/ibc-1252" }
+tendermint-light-client = { git = "https://github.com/informalsystems/tendermint-rs", branch = "thane/ibc-1252" }
+tendermint-testgen      = { git = "https://github.com/informalsystems/tendermint-rs", branch = "thane/ibc-1252" }
```

Comment:
Note that this currently uses the code from here: https://github.com/informalsystems/tendermint-rs/pull/960

Once we're happy with that PR and this one, we can merge https://github.com/informalsystems/tendermint-rs/pull/960 and update these branches to `master` until we can release tendermint-rs v0.22.0.